# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1296515967 18000
#      Mon Jan 31 18:19:27 2011 -0500
# Branch release-3-4-x
# Node ID a3c6d0480c7c4be6b4bc1e9d777d5a7e74d610a9
# Parent  1c4cad056b7981471ea3d82b137522c58b3affd5
plot/private/__axis_label__.m: don't call __fltk_redraw__

diff --git a/scripts/ChangeLog b/scripts/ChangeLog
--- a/scripts/ChangeLog
+++ b/scripts/ChangeLog
@@ -1,8 +1,12 @@
+2011-01-31  John W. Eaton  <jwe@octave.org>
+
+	* plot/private/__axis_label__.m: Don't call __fltk_redraw__.
+
 2011-01-31  John W. Eaton  <jwe@octave.org>
 
 	* plot/__fltk_ginput__.m: Remove line that should have been
 	removed in previous change.
 
 2010-01-31  Rik  <octave@nomad.inbox5.com>
 
 	* plot/private/__gnuplot_has_terminal__.m: Simplify regular expression
diff --git a/scripts/plot/private/__axis_label__.m b/scripts/plot/private/__axis_label__.m
--- a/scripts/plot/private/__axis_label__.m
+++ b/scripts/plot/private/__axis_label__.m
@@ -33,18 +33,16 @@ function retval = __axis_label__ (caller
     set (h, "fontangle", get (ca, "fontangle"),
          "fontname", get (ca, "fontname"),
          "fontsize", get (ca, "fontsize"),
          "fontunits", get (ca, "fontunits"),
          "fontweight", get (ca, "fontweight"),
          "string", txt,
          varargin{:});
 
-    __fltk_redraw__();
-
     if (nargout > 0)
       retval = h;
     endif
   else
     error ("%s: expecting first argument to be character string", caller);
   endif
 
 endfunction
